README.md

```diff
@@ -1,25 +1,112 @@
-  
-
-### Place any suggestions/problems in [issues](https://github.com/Rnen/AdminToolbox/issues)!
-
-# Thanks & Enjoy.
+
+<br>
+
+<div align = center>
+
+[![Badge Release]][Releases]   
+[![Badge Downloads]][Releases]   
+[![Badge Commit]][Commit]   
+[![Badge Watchers]][Watchers]
+
+<br>
+<br>
+
+# AdminToolbox
+*A server plugin for **[SCP:Secret Laboratory]**.*
+
+<br>
+
+---
+
+[![Button Installation]][Wiki]   
+[![Button Issues & Ideas]][Issues]   
+[![Button Development]][Projects]
+
+---
+
+</div>
+
+<br>
+<br>
+
+## Features
+
+-   Break doors that have <br>
+    been interacted with.
+
+-   Teleporting Players
+
+-   Endless Rounds
+
+-   Warp Points
+
+
+<br>
+
+### Friendly Fire
+
+*For every act of friendly fire, the server* <br>
+*console and the log files will take note.*
+
+<br>
+
+### Advanced GodMode
+
+- Turn off damage for players
+
+- Instakill
+
+*'Advanced' in the sense of `106` with ATGOD* <br>
+*cannot be contained, amongst other things.*
+
+<br>
+
+### Player Statistics
+
+*Log stats for every player to a file.*
+
+<br>
+
+### Join Message
+
+*The server window shows a message* <br>
+*for every player that joins the server.*
+
+
+<br>
+<br>
+
+## Licensing
+
+**SCP:SL** and it's development team have been <br>
+granted permission under the `CC-BY` license.
+
+<br>
+
+
+<!----------------------------------------------------------------------------->
+
+[Releases]: https://GitHub.com/Rnen/AdminToolbox/releases/
+[Watchers]: https://GitHub.com/Rnen/AdminToolbox/watchers/
+[Projects]: https://GitHub.com/Rnen/AdminToolbox/projects/1
+[Issues]: https://GitHub.com/Rnen/AdminToolbox/issues
+[Commit]: https://GitHub.com/Rnen/AdminToolbox/commit
+[Wiki]: https://GitHub.com/Rnen/AdminToolbox/wiki
+
+[SCP:Secret Laboratory]: https://store.steampowered.com/app/700330/SCP_Secret_Laboratory/
+
+
+<!--------------------------------[ Badges ]----------------------------------->
+
+[Badge Downloads]: https://img.shields.io/github/downloads/Rnen/AdminToolbox/total.svg?style=for-the-badge&color=A9225C&logoColor=white&logo=DocuSign
+[Badge Watchers]: https://img.shields.io/github/watchers/Rnen/AdminToolbox.svg?style=for-the-badge&label=Watcher&maxAge=2592000&color=73398D&logoColor=white&logo=Git
+[Badge Release]: https://img.shields.io/github/v/release/Rnen/AdminToolbox?style=for-the-badge
+[Badge Commit]: https://img.shields.io/github/last-commit/Rnen/AdminToolbox?style=for-the-badge&color=258AAF&logoColor=white&logo=GitHub
+
+
+<!-------------------------------[ Buttons ]----------------------------------->
+
+[Button Issues & Ideas]: https://img.shields.io/badge/Issues_&_Ideas-00A8E1?style=for-the-badge&logoColor=white&logo=AskUbuntu
+[Button Installation]: https://img.shields.io/badge/Installation-569A31?style=for-the-badge&logoColor=white&logo=DocuSign
+[Button Development]: https://img.shields.io/badge/Projects-66459B?style=for-the-badge&logoColor=white&logo=AzureArtifacts
+
```

Comment:
Thanks for the PR! I don't get many of those, especially nowadays...
Just got a few suggested changes to your version here
```suggestion
<br>

<div align = center>

[![Badge Release]][Releases]   
[![Badge Downloads]][Releases]   
[![Badge Commit]][Commit]   
[![Badge Watchers]][Watchers]

<br>

# AdminToolbox
*A SMod server plugin for **[SCP: Secret Laboratory]**.*

---

[![Button Installation]][Wiki]   
[![Button Issues & Ideas]][Issues]   
[![Button Development]][Projects]

---

</div>

<br>
<br>

## Features

-   Break, lock & unlock doors <br>
    upon interaction.

-   Teleporting Players

-   Endless Rounds

-   Warp Points

-   Ghost Mode

-   Instantkill

-   Infinite items

-   Jailing players

-   Server statistics


<br>

### Friendly Fire

*For every act of friendly fire, the server* <br>
*console and the log files will take note.* <br><br>
*The Plugin also supports turning off certain* <br>
*types of damage between players*

<br>

### Advanced GodMode

- Turn off damage for players

<sub>*'Advanced' in the sense of `106` with ATGOD* <br>
*cannot be contained, amongst other things.*</sub>

<br>

### Player Statistics

*Keeps track of player statistics*<br>
*like kills, deaths, team-kills & playtime*<br>
*and stores it in dedicated player files*

<br>

### Join Message

*The server window shows a player information* <br>
*for every player that joins the server.*


<br>
<br>

## Licensing

**SCP:SL** and it's development team have been <br>
granted permission under the `CC-BY` license.

<br>


<!----------------------------------------------------------------------------->

[Releases]: https://GitHub.com/Rnen/AdminToolbox/releases/
[Watchers]: https://GitHub.com/Rnen/AdminToolbox/watchers/
[Projects]: https://GitHub.com/Rnen/AdminToolbox/projects/1
[Issues]: https://GitHub.com/Rnen/AdminToolbox/issues
[Commit]: https://GitHub.com/Rnen/AdminToolbox/commit
[Wiki]: https://GitHub.com/Rnen/AdminToolbox/wiki

[SCP: Secret Laboratory]: https://store.steampowered.com/app/700330/SCP_Secret_Laboratory/


<!--------------------------------[ Badges ]----------------------------------->

[Badge Downloads]: https://img.shields.io/github/downloads/Rnen/AdminToolbox/total.svg?style=for-the-badge&color=A9225C&logoColor=white&logo=DocuSign
[Badge Watchers]: https://img.shields.io/github/watchers/Rnen/AdminToolbox.svg?style=for-the-badge&label=Watcher&maxAge=2592000&color=73398D&logoColor=white&logo=Git
[Badge Release]: https://img.shields.io/github/v/release/Rnen/AdminToolbox?style=for-the-badge
[Badge Commit]: https://img.shields.io/github/last-commit/Rnen/AdminToolbox?style=for-the-badge&color=258AAF&logoColor=white&logo=GitHub


<!-------------------------------[ Buttons ]----------------------------------->

[Button Issues & Ideas]: https://img.shields.io/badge/Issues_&_Ideas-00A8E1?style=for-the-badge&logoColor=white&logo=AskUbuntu
[Button Installation]: https://img.shields.io/badge/Installation-569A31?style=for-the-badge&logoColor=white&logo=DocuSign
[Button Development]: https://img.shields.io/badge/Projects-66459B?style=for-the-badge&logoColor=white&logo=AzureArtifacts
```

Reply:
> I don't get many of those, especially nowadays...

How unfortunate~

> They are a bit all over the place, so sorry for not submitting suggested changes per line.

Well I literally had no idea where the changes were.

*I copied the changes and had to manually re-add all*
*space and newlines to even see the formatting again.*

It seems you only changed feature descriptions, is this correct?

Reply:
> It seems you only changed feature descriptions, is this correct?

Yes, and I moved around a couple of line breaks here and there... :sweat_smile:

Reply:
I looked through them and fixed them up.